playbooks/clean.yml

```diff
@@ -31,3 +31,25 @@
       tags:
         - bor
         - clean
+
+    - name: Stop services
+      service:
+        name: "{{ item }}"
+        state: stopped
+      with_items:
+        - bor
+        - heimdalld
+        - heimdalld-bridge
+        - heimdalld-rest-server
+      become: yes
+
+    - name: Clean services file
+      file:
+        path: "/etc/systemd/system/{{ item }}"
+        state: absent
+      with_items:
+        - bor.service
+        - heimdalld.service
+        - heimdalld-bridge.service
+        - heimdalld-rest-server.service
+      become: yes
```

Comment:
Missing New line here

Reply:
@vcastellm Which line do you mean?

Reply:
```suggestion
      become: yes

```

Reply:
Empty new line at the end of the file, GH is showing the red sign